Track Your Reps/Lib/ResponseStubs.swift

```diff
@@ -12,7 +12,9 @@ public func stubbedResponse(_ stub: ResponseStub) -> Data {
 
 public enum ResponseStub: String {
     
-    case votesForMember = "{\"status\": \"OK\", \"copyright\": \"Pro Publica\", results: [{\"member_id\": \"1\", \"total_votes\": \"100\", \"offset\": \"0\", \"votes\": [{\"member_id\": \"1\", \"chamber\": \"Senate\", \"congress\": \"115\", \"session\": \"1\", \"roll_call\": \"1\", \"bill\": {\"number\": \"Res. 37\", \"bill_uri\": \"api.propublica.org/congress/v1/115/bills/hjres37\", \"title\": \"Hello title hello\", \"latest_action\": \"So much action\"}, \"member_id\": \"1\", \"description\": \"Something something something.\", \"question\": \"Is a question a question?\", \"date\": \"2017-03-02\", \"time\": \"0:00\", \"position\": \"Yes\"}]}]}"
+    case bill = "{}"
```

Comment:
**Todo**

add responseStub for bill and add a helper to the API wrapper so these fire when working offline allowing us to work w/o internet